static/src/javascripts/projects/commercial/modules/prebid/bid-config.spec.js

```diff
@@ -674,3 +674,39 @@ describe('triplelift adapter', () => {
         });
     });
 });
+
+describe('xaxis adapter', () => {
+    beforeEach(() => {
+        resetConfig();
+        config.set('page.contentType', 'Article');
+        config.set('switches.prebidXaxis', true);
+        shouldIncludeXaxis.mockReturnValue(true);
+    });
+
+    afterEach(() => {
+        jest.resetAllMocks();
+    });
+
+    test('should include xaxis adapter if condition is true ', () => {
+        expect(getBidders()).toEqual(['ix', 'xhb']);
+    });
+
+    test('should return correct xaxis adapter params for top-above-nav', () => {
```

Comment:
Would suggest expanding this test to cover all ad slots, including a case that will return NO_MATCH_ID to improve test coverage.